### PR TITLE
Update the definition of IoULoss in iou_loss docs

### PR DESCRIPTION
**Pull request by Anirban0011**
## What this PR does:

* Attempts to expand upon the existing definition of IoULoss

### CODE REVIEW

**Comment by Anirban0011** [comment]
@ianstenbit @LukeWood , kindly review whenever possible. Thanks!

File: keras_cv/losses/iou_loss.py [hunk 1]
```diff
@@ -23,11 +23,13 @@ class IoULoss(keras.losses.Loss):
     """Implements the IoU Loss
 
     IoU loss is commonly used for object detection. This loss aims to directly
```
Comment (ianstenbit) [inline keras_cv/losses/iou_loss.py:25]: How about instead we just say "IoU (Intersection over Union) loss ..." and scrap the other changes?

**Review by ianstenbit — CHANGES_REQUESTED** [review]
